presto-raptor/src/test/java/com/facebook/presto/raptor/systemtables/TestShardMetadataRecordCursor.java

```diff
@@ -54,26 +58,34 @@
 import static com.facebook.presto.spi.predicate.Range.lessThanOrEqual;
 import static com.facebook.presto.spi.type.BigintType.BIGINT;
 import static com.facebook.presto.spi.type.DateType.DATE;
+import static com.facebook.presto.spi.type.IntegerType.INTEGER;
+import static com.facebook.presto.spi.type.TimestampType.TIMESTAMP;
 import static com.facebook.presto.spi.type.VarcharType.createVarcharType;
 import static com.facebook.presto.testing.MaterializedResult.DEFAULT_PRECISION;
 import static com.facebook.presto.testing.TestingConnectorSession.SESSION;
 import static io.airlift.slice.Slices.utf8Slice;
+import static java.lang.String.format;
+import static java.util.concurrent.TimeUnit.DAYS;
 import static java.util.stream.Collectors.toList;
 import static org.testng.Assert.assertEquals;
 
 @Test(singleThreaded = true)
 public class TestShardMetadataRecordCursor
 {
     private static final SchemaTableName DEFAULT_TEST_ORDERS = new SchemaTableName("test", "orders");
+    private static final int DEFAULT_TEST_ORDERS_ID = 1;
+    private static final SchemaTableName TIMESTAMP_TABLE = new SchemaTableName("test2", "timestamp");
+    private static final int TIMESTAMP_TABLE_ID = 2;
 
     private Handle dummyHandle;
     private ConnectorMetadata metadata;
-    private IDBI dbi;
+    private DBI dbi;
 
     @BeforeMethod
     public void setup()
     {
         this.dbi = new DBI("jdbc:h2:mem:test" + System.nanoTime());
+        this.dbi.registerMapper(new TableColumn.Mapper(new TypeRegistry()));
```

Comment:
In the code, we register parameterized JDBI mapper in the `RaptorModule` using guice. In the unit tests we need to do this manually. Since we didn't retrieve column information in `ShardmetadataRecordCursor` before, the previous test code didn't need that `registerMapper` function call.